views/courses/index.js

```diff
@@ -0,0 +1,83 @@
+const {coursesModel} = require('../../models/courses');
```

Comment:
everything here is a controller layer


Model
   - data access
   - dto

Controller
   - request/response lifecycle
   - model sey baat karna
   - view sey baat karna
   - send response back

res.render('courses/create', ({message: 'coursed added successfully', errors: null}));
    - create html out of data (2nd param) < VIEW
    - send html to response


thin controller
 - delegate all stuff into models / views


view and models
   - they have no knowledge of req + res
   - model layer > API? DB? cache? email?
   -   req/res red flag
   -

Reply:
GET /courses
   - HTML (view) . res.render
   - JSON (view) . res.json
   - PDF (view). pdf